### PR TITLE
10153-Button-alignment-text-cutting-off-Vehicle-table-scrolling-updates

### PR DESCRIPTION
**Pull request by ZaidGhauri**
Issue #:/bcgov/entity/issues/10153

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/assets/styles/overrides.scss [hunk 1]
```diff
@@ -339,10 +339,18 @@ input#reg-textfield.v-text-field.v-input--dense:not(.v-text-field--outlined),
...
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
 .registration-table.v-data-table tr.filter-row td {
   padding-bottom: 0px !important;
 }
 
+button.smaller-button.pr-4 {padding-right: 0 !important;}
```
Comment (kialj876) [inline ppr-ui/src/assets/styles/overrides.scss]: I think you should be able to set this in the html with pr-0 ?

**Review by kialj876 — COMMENTED** [review]
Hey Zaid, looks good to me. I just have one comment / question for ya

**Review by doug-lovett — APPROVED** [review]
Looks good: no comment to add to Kial's.

**Review by jordiwes — APPROVED** [review]
LGTM

**Review by kialj876 — APPROVED** [review]

**Pull request merged**